readme.rst

```diff
@@ -3,43 +3,63 @@ Documentation starter pack
 
 See the `Sphinx and Read the Docs <https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/>`_ guide for instructions on how to get started with Sphinx documentation.
 
-Then go through the following sections to use this starter pack to set up your documentation repository.
+Then go through the following sections to use this starter pack to set up your docs repository.
 
-Prerequisites
--------------
-
-Before you begin, ensure you have the following:
+Set up your documentation repository
+------------------------------------
 
-* A GitHub repository where you want to host your documentation, cloned to your local machine. The recommended approach is to host the documentation alongside your code in a `docs` folder. But a standalone documentation repository is also an option; in this case, start with an empty repository.
-* Git and Bash installed on your system.
+You can either create a standalone documentation project based on this repository or include the files from this repository in a dedicated documentation folder in an existing code repository.
 
-Initialise your documentation repository
----------------------------------------
+**Note:** We're planning to provide the contents of this repository as an installable package in the future, but currently, you need to copy and update the required files manually.
```

Comment:
Can we move this admonishment above this section's header? It seem out of place in its current location.

Reply:
I think this review is outdated?

Reply:
In this PR's build, the admonishment is being displayed. I think it should be moved one level up as it provides very general information about the starter pack. It should probably also be reworded:

**Note:** Once a user has enabled their repository with the starter pack, they will need to stay in sync by tracking changes and making the necessary updates manually. The intention is to eventually provide the starter pack as a package in order to make updates easier.

Reply:
I agree with the proposed change but I suggest we do it in a different PR.

I see the admonishment under [Set up your documentation repository](https://canonical-starter-pack--87.com.readthedocs.build/readme/#set-up-your-documentation-repository) - but looks like this is the same in the published starter pack's Readme, again under [Set up your documentation repository](https://canonical-starter-pack.readthedocs-hosted.com/readme/#set-up-your-documentation-repository).

So from what I can see this isn't a change introduced here. So it is more suitable to do it in a separate PR.

Reply:
I agree that this is a different task. The idea was to tell people that "sorry, setup is complicated at the moment", that's why it is under the Setup section. But we can definitely expand on this.